modules/data-entry/src/main/frontend/src/dataHomepage/UserDashboard.jsx

```diff
@@ -16,163 +16,164 @@
-                        {questionnaire["title"]}
-                      </Button>
-                    </Link>
-                  }
-                  action={
-                    <NewFormDialog presetPath={questionnaire["@path"]}>
-                      New form
-                    </NewFormDialog>
-                  }
-                  classes={{
-                    action: classes.newFormButtonHeader
-                  }}
-                />
-                <CardContent>
-                  <LiveTable
-                    columns={columns}
-                    customUrl={customUrl}
-                    defaultLimit={10}
-                    joinChildren="lfs:Answer"
-                    questionnaire={questionnaire["@path"]}
-                    entryType={questionnaire["title"]}
-                    filters
-                    actions={actions}
-                    />
-                </CardContent>
-              </Card>
+        {
+          dashboardExtensions.map((extension) => {
+            let Extension = extension["lfs:extensionRender"];
+            return <Grid item lg={12} xl={6}>
+              <Extension />
             </Grid>
-          )
-        })}
+          })
+        }
       </Grid>
+      <Dialog open={open} onClose={() => setOpen(false)}>
+        <DialogTitle>
+          <Typography variant="h6" className={classes.dialogTitle}>Add</Typography>
+        </DialogTitle>
+        <DialogContent>
+          <MaterialTable
+            columns={[
+              { title: 'Type', field: 'lfs:extensionName' },
+            ]}
+            data={creationExtensions}
+            options={{
+              search: true,
+              pageSize: rowCount,
+              showTitle: false,
+              rowStyle: rowData => ({
+                // /* It doesn't seem possible to alter the className from here */
+                backgroundColor: (selectedRow && selectedRow["jcr:uuid"] === rowData["jcr:uuid"]) ? theme.palette.grey["200"] : theme.palette.background.default
+              })
+            }}
+            onRowClick={(event, rowData) => {
+              setSelectedRow(rowData);
+            }}
+            onChangeRowsPerPage={pageSize => {
+              setRowCount(pageSize);
+            }}
+            />
+        </DialogContent>
+        <DialogActions>
+          <Button
+            variant="contained"
+            color="default"
+            onClick={() => setOpen(false)}
+            >
+            Cancel
+          </Button>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={ () => {
+              setOpen(false);
+              setSelectedCreation(creationExtensions.indexOf(selectedRow));
+            }}
+            >
+            Next
+          </Button>
+        </DialogActions>
+      </Dialog>
+      <div className={classes.newFormButtonWrapper}>
+        <Tooltip title={"Add"} aria-label="add">
```

Comment:
"Add" is fine, "New" would also work and be consistent with the "New subject", "New form" tooltips.
```suggestion
        <Tooltip title={"New"} aria-label="add">
```